src/codegen/luacg.cpp

```diff
@@ -949,7 +958,7 @@ wxString LuaCodeGenerator::GetConstruction(PObjectBase obj, bool silent, wxStrin
 		strTableName = table->GetValueAsString();
 		if (strTableName.empty())
 		{
-			strTableName = wxT("UI");
+			strTableName = "self";
```

Comment:
If the table name is not empty, its value will be used, not this new default value